timemachine/parallel/utils.py

```diff
@@ -8,7 +8,10 @@
 
 def get_gpu_count() -> int:
     # Expected to return a line delimited summary of each GPU
-    output = check_output(["nvidia-smi", "-L"])
+    try:
+        output = check_output(["nvidia-smi", "-L"])
+    except FileNotFoundError:
```

Comment:
Perhaps a log statement after this? Didn't want it to be noisy, but can see an argument for one.

Reply:
Driver errors will still raise an exception. This only returns 0 if there is no binary for `nvidia-smi`